### PR TITLE
TH-98 select component

### PR DESCRIPTION
**Pull request by Onibenjo**
This PR aims to:
- add Select component
- abstract the custom dropdown paths to a variable

### CODE REVIEW

File: src/components/Select/Select.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Select } from '../index';
+import { fireEvent, render, screen, within } from '@testing-library/react';
+
+describe('Select', () => {
+  it('should trigger the on change function when input value is changed', () => {
+    const onChangeMock = jest.fn();
+    render(
+      <>
```
Comment (cingopablo) [inline src/components/Select/Select.spec.tsx:9]: Why do we need a fragment there?

**Review by cingopablo — APPROVED** [review]

**Comment by milatimber** [comment]
Only comment from my side: would be cool to have `required` possibility for options, as we discussed.
Other than that - great job!

File: src/components/Select/Select.spec.tsx (hunk 1 above)
Comment (Onibenjo) [reply src/components/Select/Select.spec.tsx:9]: good point. missed that

**Comment by Onibenjo** [comment]
> Only comment from my side: would be cool to have `required` possibility for options, as we discussed. Other than that - great job!

Noted.. Looking into it

**Review by cingopablo — APPROVED** [review]

**Review by milatimber — APPROVED** [review]
LGTM! great job!

**Pull request merged**